spec/jobs/reports/drop_off_report_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+RSpec.describe Reports::DropOffReport do
+  let(:report_date) { Date.new(2023, 12, 12).in_time_zone('UTC') }
+  let(:report_config) do
+    '[{"email":"ursula@example.com",
+       "issuers":"urn:gov:gsa:openidconnect.profiles:sp:sso:agency_name:app_name"}]'
+  end
+
+  before do
+    allow(IdentityConfig.store).to receive(:drop_off_report_config).and_return(report_config)
+  end
+
+  describe '#perform' do
+    it 'gets a CSV from the report maker, and sends email' do
+      # rubocop:disable Layout/LineLength
+      reports = Reporting::EmailableReport.new(
+        title: 'Drop Off Report',
+        table: [
+          ['Term', 'Description', 'Definition', 'Calculated'],
```

Comment:
since we're not checking accuracy tables of the report here, can we make this smaller?

```ruby
  title: 'Drop Off Report',
  table: [
    ['heading1', 'heading2'],
    [1,2]
  ],
filename: 'drop_off_report',
```

Reply:
Truncated.